upb/decode_fast.c

```diff
@@ -52,8 +52,9 @@
 
 #define UPB_PARSE_ARGS d, ptr, msg, table, hasbits, data
 
-#define RETURN_GENERIC(m)  \
+#define RETURN_GENERIC(m)   \
   /* fprintf(stderr, m); */ \
+  /*__builtin_trap(); */    \
```

Comment:
I am confused, why does this macro expand to commented out code?

Reply:
The meat of the macro is the call to `fastdecode_generic()`, these two commented-out lines are just there for convenience, to un-comment during debugging. It's the only reason that the `m` parameter to the macro exists, so we can print it out during debugging.

If you prefer I can remove the commented-out code, or add a comment to explain why it is there.

Reply:
a comment is fine

Reply:
Done.